homeassistant/components/zha/binary_sensor.py

```diff
@@ -285,6 +285,18 @@ class XiaomiPlugConsumerConnected(BinarySensor):
     _attr_translation_key: str = "consumer_connected"
 
 
+@MULTI_MATCH(
+    cluster_handler_names="tuya_manufacturer",
+    quirk_classes={"zhaquirks.tuya.ts0601_trv.ZonnsmartTV01_ZG"},
+)
+class ZonnSmartWindowOpened(BinarySensor):
+    """Sensor that display if the window is detected open or closed."""
+
+    _attribute_name = "window_detection"
+    _unique_id_suffix = "window_detection"
+    _attr_translation_key: str = "window_detection"
```

Comment:
Is this entity even created when freshly pairing a TRV? (So, delete the TRV completely, then pair it again)
`window_detection` is not initialized from what I can see.

It should likely be initialized here from what I can see: https://github.com/home-assistant/core/blob/dev/homeassistant/components/zha/core/cluster_handlers/manufacturerspecific.py